src/main/java/edu/ucsb/cs156/kitchensink/controllers/UsersController.java

```diff
@@ -0,0 +1,37 @@
+package edu.ucsb.cs156.kitchensink.controllers;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import edu.ucsb.cs156.kitchensink.entities.User;
+import edu.ucsb.cs156.kitchensink.repositories.UserRepository;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Api(description="User information (admin only)")
+@RequestMapping("/api/admin/users")
+@RestController
+public class UsersController extends ApiController {
+    @Autowired
+    UserRepository userRepository;
+
+    @Autowired
+    ObjectMapper mapper;
+
+    @ApiOperation(value = "Get a list of all users")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/")
```

Comment:
Try to be consistent between this and the `@GetMapping("")` in `src/main/java/edu/ucsb/cs156/kitchensink/controllers/UserInfoController.java`